src/main/java/com/alex/eshop/eshop/Service/ItemServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.alex.eshop.eshop.Service;
+
+import com.alex.eshop.eshop.Entity.Category;
+import com.alex.eshop.eshop.Entity.Item;
+import com.alex.eshop.eshop.Repository.ItemRepository;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+import javax.transaction.Transactional;
+import java.util.*;
+
+@Service
+@Component
+@Transactional
+public class ItemServiceImpl implements ItemService {
+    @PersistenceContext private EntityManager entityManager;
```

Comment:
Remove `@PersistenceContext private EntityManager entityManager;` if not going to use